plugin/utility/src/main/java/org/opencds/cqf/ruler/plugin/utility/ClientUtilities.java

```diff
@@ -19,6 +19,28 @@
  * setting up authentication
  */
 public interface ClientUtilities {
+    public static String SERVER_URL = "http://localhost:%d/fhir";
+
+        /**
+     * Creates an client url given a url template and port
+     * 
+     * @param theUrlTemplate    the url template to use
+     * @param thePort           the port to use
+     * @return String for the client url
+     */
+    public default String getClientUrl(String theUrlTemplate, Integer thePort) {
```

Comment:
This code isn't used anywhere?

Reply:
It's used in the ReportProviderIT.  I thought it was going to be needed in non test classes too, but looks like that didn't pan out.

It could probably be moved to a Report utilities.

Reply:
Moved to a test-utility plugin.